packages/mds-transaction-api/spec/spec.yaml

```diff
@@ -0,0 +1,334 @@
+openapi: "3.0.2"
+info:
+  title: MDS Transaction API
+  version: "0.0.1"
```

Comment:
Isn't the starter version number usually `0.1.0`?

Reply:
Yeah I suppose so, will update